spec/controllers/teams_controller_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe TeamsController, type: :controller do
```

Comment:
You can also open up index.html in coverage folder to see which lines of your methods are not covered.